incubator/minecraft/templates/NOTES.txt

```diff
@@ -0,0 +1,46 @@
+Getting Started:
```

Comment:
Can you remove this line? It's redundant since the header helm prints out is `NOTES:`.